composer.json

```
@@ -50,7 +51,9 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Tests\\Unit\\Neusta\\Pimcore\\TestingFramework\\": "tests/Unit"
+      "Fixtures\\": "tests/Fixtures/",
+      "Tests\\Functional\\Neusta\\Pimcore\\TestingFramework\\": "tests/Functional/",
+      "Tests\\Unit\\Neusta\\Pimcore\\TestingFramework\\": "tests/Unit/"
```

Ich hätte erwartet, dass das so aussieht:
```suggestion
      "Neusta\\Pimcore\\TestingFramework\\Tests\\": "tests/",
```
Das Function, Unit und Fixtures alles Ordner mit Großbuchstaben sind, sollten die alle unter den Namespace fallen. Allerdings wäre dafür natürlich refactoring der Test Klassen notwendig.

---

You are right. I fixed it.